src/components/templates/Playa/VenuePreview.tsx

```diff
@@ -1,7 +1,11 @@
 import React, { useEffect, useState } from "react";
 import { FirebaseReducer } from "react-redux-firebase";
-import { Venue, VenuePlacementState } from "types/Venue";
-import { VenueEvent } from "types/VenueEvent";
+import {
+  Venue,
+  VenueEvent,
+  VenuePlacementState,
+  VenueTemplate,
+} from "types/venues";
```

Comment:
I thought Playa is completely removed from our codebase?

Reply:
It will be in the PR that removes it, which is separate to this PR, and will be merged after this PR. Both have been hanging around for ages, and this PR is the blocker (in my mental ordering/desire to complete things) of me working on the ‘delete all the old things’ PR.